Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h

```diff
@@ -13,82 +13,225 @@ NS_ASSUME_NONNULL_BEGIN
-//defaults to YES
+/**
+ Whether to deselect the selected cell of the table view after sending the selected object to the
+ delegate. Defaults to YES.
+ */
 @property BOOL clearsTableViewCellSelection;
 
+/**
+ Accessor for the fetchedObjects array on the fetched results controller
+
+ @return Array of fetched NSManagedObjects
+ */
 - (nullable VOKArrayOfManagedObjects *)fetchedObjects;
 
+/**
+ Reload the fetcehd data in the fetched results controller, which will cause a table view reload.
+ */
 - (void)reloadData;
-- (BOOL)reloadData:(NSError **)error;
 
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-              tableView:(nullable UITableView *)tableView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-              tableView:(nullable UITableView *)tableView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-              batchSize:(NSInteger)batchSize;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-              tableView:(nullable UITableView *)tableView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-              tableView:(nullable UITableView *)tableView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-              batchSize:(NSInteger)batchSize
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-              tableView:(nullable UITableView *)tableView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-              batchSize:(NSInteger)batchSize
-             fetchLimit:(NSInteger)fetchLimit
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
+/**
+ Reload the fetcehd data in the fetched results controller, which will cause a table view reload.
```

Comment:
misspelled here as well. maybe you should just stop trying to make fetch happen...